packages/salesforcedx-vscode-core/src/commands/forceSourceDeployManifest.ts

```diff
@@ -36,6 +42,40 @@ export class ForceSourceDeployManifestExecutor extends BaseDeployExecutor {
   }
 }
 
+export class LibrarySourceDeployManifestExecutor extends LibraryCommandletExecutor<string> {
+  protected logName = 'force_source_deploy_with_manifest';
+  protected executionName = 'Deploy With Manifest (beta)';
+
+  protected async run(response: ContinueResponse<string>): Promise<boolean> {
+    try {
```

Comment:
We should add a catch condition so we can post the error to the output panel, etc

Reply:
nevermind, you just reminded me we handle that in the `execute` method of `LibraryCommandletExecutor`